sdk/tables/azure-data-tables/samples/sample_copy_table.py

```diff
@@ -0,0 +1,110 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_copy_table.py
+
+DESCRIPTION:
+    These samples demonstrate how to copy tables between table and blob storage.
+
+USAGE:
+    python sample_copy_table.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_STORAGE_CONNECTION_STRING - the connection string to your storage account
+    2) TABLES_STORAGE_ENDPOINT_SUFFIX - the Table service account URL suffix
+    3) TABLES_STORAGE_ACCOUNT_NAME - the name of the storage account
+    4) TABLES_PRIMARY_STORAGE_ACCOUNT_KEY - the storage account access key
+"""
+
+
+from datetime import datetime
+from uuid import uuid4
+import os
+import json
+from azure.storage.blob import BlobServiceClient
+from azure.data.tables import TableClient
+from dotenv import find_dotenv, load_dotenv
+
+
+class CopyTableSamples(object):
+    def __init__(self):
+        load_dotenv(find_dotenv())
+        self.access_key = os.getenv("TABLES_PRIMARY_STORAGE_ACCOUNT_KEY")
+        self.endpoint_suffix = os.getenv("TABLES_STORAGE_ENDPOINT_SUFFIX")
+        self.account_name = os.getenv("TABLES_STORAGE_ACCOUNT_NAME")
+        self.table_connection_string = "DefaultEndpointsProtocol=https;AccountName={};AccountKey={};EndpointSuffix={}".format(
+            self.account_name, self.access_key, self.endpoint_suffix
+        )
+        self.table_client = TableClient.from_connection_string(conn_str=self.table_connection_string, table_name="mytable")
```

Comment:
I would just store the TableServiceClient - then use it to create the TableClient in each of the sample methods below. That way we can recreate the TableClient using the name of the container.